dashboard/src/components/AppView/AppView.tsx

```diff
@@ -129,23 +134,42 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     const dropByName = (array: Array<IKubeItem<IResource>>) => {
       return _.dropWhile(array, r => r.item && r.item.metadata.name === resource.metadata.name);
     };
+    let apiVersion = "v1";
+    let apiResource: string;
     switch (resource.kind) {
       case "Deployment":
         const newDeps = dropByName(this.state.deployments);
         newDeps.push(newItem);
         this.setState({ deployments: newDeps });
+        // We won't use resource.apiVersion as it may have a different version
```

Comment:
I am not sure I understand this comment, why don't we want to use the `apiVersion` in the cache?

Reply:
So I need to do some testing here, my concern is that the API server will convert, for example an `extensions/v1beta1` to an `apps/v1` version for Deployments if available. So a chart manifest with an older API version will lead to inconsistencies. I think we need to fix the API version for fetching Deployments to `apps/v1` regardless of what the server returns or is in the release manifest to avoid this inconsistency.

Reply:
On further testing, the API server doesn't actually change the apiVersion field on the object, so I'm able to consistently get `extensions/v1beta` back from the API. Given this, I think we can change this to use the `apiVersion` from the returned resource, though we should keep this in mind in case it can cause issues in the future.

Reply:
+1 thanks for checking (BTW you forgot to remove the comment?)

Reply:
done, thanks!